apps/prairielearn/src/tests/questionSharing.test.ts

```diff
@@ -224,17 +225,16 @@ describe('Question Sharing', function () {
           unsafe_course_sharing_token: testCourseSharingToken,
         }),
       });
-
       assert(res.ok);
-      const sharingPage = await fetch(sharingPageUrl(testCourseId));
+
+      const sharingPage = await fetchCheerio(sharingPageUrl(exampleCourseId));
       assert(sharingPage.ok);
-      const sharingPageText = await sharingPage.text();
-      assert(sharingPageText.includes('XC 101'));
+      assert.include(sharingPage.$('[data-testid="shared-with"]').text(), 'QA 101');
```

Comment:
This test was actually fetching and asserting the wrong thing. It just so happened to pass because the string "XC 101" appeared elsewhere in the HTML document.